libfreerdp/codec/clear.c

```diff
@@ -82,6 +82,35 @@ static const UINT32 CLEAR_LOG2_FLOOR[256] = {
 
 static const BYTE CLEAR_8BIT_MASKS[9] = { 0x00, 0x01, 0x03, 0x07, 0x0F, 0x1F, 0x3F, 0x7F, 0xFF };
 
+void clear_reset_vbar_storage(CLEAR_CONTEXT* clear)
+{
+	int i;
+
+	for (i = 0; i < 32768; i++)
```

Comment:
`ARRAYSIZE(clear->VBarStorage)` might be better here instead of the magic number